### PR TITLE
TRN-341: Comments and search page

### PR DESCRIPTION
**Pull request by debjyotisaha13**
- Added custom comment type
- Added custom search bar using Search API module.

### CODE REVIEW

File: config/node.type.news.yml [hunk 1]
```diff
@@ -14,5 +14,5 @@ type: news
 description: 'Show all the news at Innoraft'
 help: ''
 new_revision: true
-preview_mode: 0
+preview_mode: 1
```
Comment (kkmitra) [inline config/node.type.news.yml:17]: Why is this line changed?

Comment (debjyotisaha13) [reply config/node.type.news.yml:17]: Fixed

Comment (debjyotisaha13) [reply config/node.type.news.yml]: This was done to show a preview of the newly created news content.
